station.localization.json

```diff
@@ -4147,7 +4147,7 @@
       "ru": "документацию",
       "sv": "läs dokumentationen",
       "fr": "page de support",
-      "de": ""
+      "de": "Überprüfen Sie die Dokumentation",
```

Comment:
This line should be exactly a part of translation from "settings_gateway_details". This text will be highlighted as hyperlink. Probably it should be: finden Sie in der Dokumentation

![image](https://user-images.githubusercontent.com/24788772/135486530-c7644cc6-e902-4fdc-a844-17af2a31e165.png)